src/fundus/publishers/ca/__init__.py

```diff
@@ -16,3 +17,12 @@ class CA(metaclass=PublisherGroup):
             RSSFeed("https://www.cbc.ca/webfeed/rss/rss-canada"),
         ],
     )
+    TheGlobeAndMail = Publisher(
+        name="The Globe and Mail",
+        domain="https://www.theglobeandmail.com",
+        parser=TheGlobeAndMailParser,
+        sources=[
+            NewsMap("https://www.theglobeandmail.com/arc/outboundfeeds/news-sitemap-index/?outputType=xml"),
+            Sitemap("https://www.theglobeandmail.com/arc/outboundfeeds/sitemap-index/?outputType=xml"),
```

Comment:
Unfortunately this doesn't seem to be a propper `Sitemap` but just another `NewsMap` since it only dates back one week.